Cython/Compiler/Code.py

```diff
@@ -356,7 +356,22 @@ def read_utilities_from_utility_dir(path):
 # by default, read utilities from the utility directory.
 read_utilities_hook = read_utilities_from_utility_dir
 
-class UtilityCodeBase:
+
+class AbstractUtilityCode:
```

Comment:
These methods do not look like they need fast access. What's the purpose of this class in comparison to simple duck typing?